docs/examples/code-cells.ipynb

```diff
@@ -0,0 +1,960 @@
+    "\n",
+    "\n",
+    "YouTubeVideo(\"9_OIs49m56E\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Interactive Widgets (HTML only)\n",
+    "\n",
+    "The basic widget infrastructure is provided by\n",
+    "the [ipywidgets](https://ipywidgets.readthedocs.io/) module.\n",
+    "More advanced widgets are available in separate packages,\n",
+    "see for example https://jupyter.org/widgets.\n",
+    "\n",
+    "The JavaScript code which is needed to display Jupyter widgets\n",
+    "is loaded automatically (using RequireJS).\n",
+    "If you want to use non-default URLs or local files,\n",
+    "you can use the\n",
+    "[nbsphinx_widgets_path](https://nbsphinx.readthedocs.io/en/0.9.7/configuration.html#nbsphinx_widgets_path) and\n",
+    "[nbsphinx_requirejs_path](https://nbsphinx.readthedocs.io/en/0.9.7/configuration.html#nbsphinx_requirejs_path)\n",
+    "settings."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import ipywidgets as w"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "slider = w.IntSlider()\n",
+    "slider.value = 42\n",
+    "slider"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "A widget typically consists of a so-called \"model\" and a \"view\" into that model.\n",
+    "\n",
+    "If you display a widget multiple times,\n",
+    "all instances act as a \"view\" into the same \"model\".\n",
+    "That means that their state is synchronized.\n",
+    "You can move either one of these sliders to try this out:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "slider"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can also link different widgets.\n",
+    "\n",
+    "Widgets can be linked via the kernel\n",
+    "(which of course only works while a kernel is running)\n",
+    "or directly in the client\n",
+    "(which even works in the rendered HTML pages).\n",
+    "\n",
+    "Widgets can be linked uni- or bi-directionally.\n",
+    "\n",
+    "Examples for all 4 combinations are shown here:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "link = w.IntSlider(description=\"link\")\n",
+    "w.link((slider, \"value\"), (link, \"value\"))\n",
+    "jslink = w.IntSlider(description=\"jslink\")\n",
+    "w.jslink((slider, \"value\"), (jslink, \"value\"))\n",
+    "dlink = w.IntSlider(description=\"dlink\")\n",
+    "w.dlink((slider, \"value\"), (dlink, \"value\"))\n",
+    "jsdlink = w.IntSlider(description=\"jsdlink\")\n",
+    "w.jsdlink((slider, \"value\"), (jsdlink, \"value\"))\n",
+    "w.VBox([link, jslink, dlink, jsdlink])"
```

Comment:
the labels "link", "dlink", "jslink", and "jsdlink" won't communicate the 4 scenarios clearly to many readers. Can we change to something like:

- kernel vs javascript
- bidirectional vs oneway

and also specify that what these are (bi/uni)directionally linking to is `slider` from the prior cell?

Reply:
Tried to address this feedback in my latest updates!